index.bs

```diff
@@ -4123,7 +4123,7 @@ object=] for a given credential. Its format is shown in <a href="#table-attested
         <tr>
             <td><dfn>credentialIdLength</dfn></td>
             <td>2</td>
-            <td>Byte length <strong>L</strong> of Credential ID, 16-bit unsigned big-endian integer.</td>
+            <td>Byte length <strong>L</strong> of [=credentialId=], 16-bit unsigned big-endian integer. Value MUST be &lt; 1024.</td>
```

Comment:
do we want to say `&le; 1023` here also?

Reply:
(Done.)